sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.c

```diff
@@ -383,4 +426,21 @@ static inline bool _node_unreachable(_nib_onl_entry_t *node)
     }
 }
 
+uint32_t _evtimer_lookup(const void *ctx, uint16_t type)
+{
+    evtimer_msg_event_t *event = (evtimer_msg_event_t *)_nib_evtimer.events;
+    uint32_t offset = 0;
+
+    DEBUG("nib: lookup ctx = %p, type = %u\n", (void *)ctx, type);
+    while (event != NULL) {
+        offset += event->event.offset;
+        if ((event->msg.type == type) &&
+            ((ctx == NULL) || (event->msg.content.ptr == ctx))) {
+            return offset;
```

Comment:
From my naive understanding I would expect 

```C
    return event->event.offset;
```

here

Reply:
That would just give an arbritrary offset to the event of the event that fires before `event`.